charts/gsp-cluster/templates/02-gsp-system/cloudhsm-service.yaml

```
@@ -0,0 +1,88 @@
+{{ if .Values.global.cluster.cloudHSMExpose }}
```

```suggestion
{{ if .Values.cloudHSMExpose }}
```

The values.yaml https://github.com/alphagov/tech-ops-cluster-config/pull/215/files is setting the value at the top-level not in the globals bit.... so I think you want simply `.Values.cloudHSMExpose`

---

Excellent - thanks, changed as suggested.